.github/workflows/deploy.yml

```diff
@@ -0,0 +1,46 @@
+name: Deploy to GitHub Pages
+
+defaults:
```

Comment:
Defaults and other things in file were added because Docusaurus won't live in root